app/controllers/pages.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+get '/' do
+  redirect '/dashboard' if session[:email]
+  erb :index
+end
+
+get '/dashboard' do
+  require_admin
+  @user = User.first(email: session[:email])
+  @digests = @user.userdigests.reverse
+  @digest = @digests[0]
+
+  erb :dashboard
+end
+
+get '/unsubscribe' do
+  require_admin
+  @user = User.first(email: session[:email])
+  @user.unsubscribed = @user.unsubscribed? ? false : true
```

Comment:
what is the purpose of this line. It looks like a repeatition since `@user.unsubscribed?` returns a boolen too

Sidenote, for methods that return a boolean and you want to assign a boolean, there is no need for ternary operater. The method is self sufficient